### PR TITLE
Poetic edits 887

### PR DESCRIPTION
**Pull request by bradleycamacho**
This is a PR for work done about [Issue-887](https://github.com/newrelic/docs-website/issues/887).

I have created a  new doc with poetry in it, and added a quote from Kafka in another doc.

### CODE REVIEW

File: src/content/docs/agents/python-agent/getting-started/poetry-break.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: 'Poetry Break'
+contentType: page
+template: basicDoc
+---
+
+##Poem
+
+But a bird that stalks
```
Comment (mmfred) [inline src/content/docs/agents/python-agent/getting-started/poetry-break.mdx:9]: @bradleycamacho IIRC you have ordered the stanzas incorrectly. Please reverse. I also think you should include a lead-in sentence.

File: src/content/docs/agents/python-agent/getting-started/poetry-break.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+his feet are tied
+so he opens his throat to sing.
+
+A free bird leaps
+on the back of the wind
+and floats downstream
+till the current ends
+and dips his wing
+in the orange sun rays
+and dares to claim the sky.
```
Comment (mmfred) [inline src/content/docs/agents/python-agent/getting-started/poetry-break.mdx:23]: Please give the author's name.

**Review by mmfred — COMMENTED** [review]
Please take a look at these suggestions.

File: src/content/docs/agents/python-agent/getting-started/poetry-break.mdx (hunk 1 above)
Comment (bradleycamacho) [reply src/content/docs/agents/python-agent/getting-started/poetry-break.mdx:20]: Flipped these!

File: src/content/docs/agents/python-agent/getting-started/poetry-break.mdx (hunk 2 above)
Comment (bradleycamacho) [reply src/content/docs/agents/python-agent/getting-started/poetry-break.mdx]: Added her name and poem title!

**Pull request closed without merging**